cli/command/image/build.go

```diff
@@ -176,6 +178,11 @@ func NewBuildCommand(dockerCli command.Cli) *cobra.Command {
 	flags.StringArrayVar(&options.ssh, "ssh", []string{}, "SSH agent socket or keys to expose to the build (only if BuildKit enabled) (format: default|<id>[=<socket>|<key>[,<key>]])")
 	flags.SetAnnotation("ssh", "version", []string{"1.39"})
 	flags.SetAnnotation("ssh", "buildkit", nil)
+
+	flags.StringArrayVarP(&options.outputs, "output", "o", []string{}, "Output destination (format: type=local,dest=path)")
```

Comment:
did we decide on the shorthand `-o` option? (just to be sure; I'm cautious adding shorthands if other options may be added that also start with the same letter)

Reply:
`-o` is a common convention in many other build tools for output. I think it would be weird to ever use it for anything else in `docker build`